modules/oe_content_event/config/install/core.entity_view_display.node.oe_event.default.yml

```diff
@@ -0,0 +1,274 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.node.oe_event.body
+    - field.field.node.oe_event.field_description_summary
+    - field.field.node.oe_event.field_description_title
+    - field.field.node.oe_event.field_end_date
+    - field.field.node.oe_event.field_entrance_fee
+    - field.field.node.oe_event.field_event_status
+    - field.field.node.oe_event.field_event_type
+    - field.field.node.oe_event.field_event_website
+    - field.field.node.oe_event.field_featured_media
+    - field.field.node.oe_event.field_featured_media_legend
+    - field.field.node.oe_event.field_online_description
+    - field.field.node.oe_event.field_online_link
+    - field.field.node.oe_event.field_online_time_end
+    - field.field.node.oe_event.field_online_time_start
+    - field.field.node.oe_event.field_online_type
+    - field.field.node.oe_event.field_organiser_internal
+    - field.field.node.oe_event.field_organiser_name
+    - field.field.node.oe_event.field_registration_capacity
+    - field.field.node.oe_event.field_registration_end_date
+    - field.field.node.oe_event.field_registration_start_date
+    - field.field.node.oe_event.field_registration_status
+    - field.field.node.oe_event.field_registration_url
+    - field.field.node.oe_event.field_report_text
+    - field.field.node.oe_event.field_report_title
+    - field.field.node.oe_event.field_start_date
+    - field.field.node.oe_event.field_summary_report
+    - field.field.node.oe_event.oe_author
+    - field.field.node.oe_event.oe_subject
+    - field.field.node.oe_event.oe_teaser
+    - node.type.oe_event
+  module:
+    - datetime
+    - link
+    - options
+    - rdf_skos
+    - text
+    - user
+id: node.oe_event.default
+targetEntityType: node
+bundle: oe_event
+mode: default
+content:
+  body:
+    weight: 0
+    label: hidden
+    settings: {  }
+    third_party_settings: {  }
+    type: text_default
+    region: content
+  field_description_summary:
+    weight: 13
+    label: above
+    settings: {  }
+    third_party_settings: {  }
+    type: text_default
+    region: content
+  field_description_title:
+    weight: 14
+    label: above
+    settings:
+      link_to_entity: false
+    third_party_settings: {  }
+    type: string
+    region: content
+  field_end_date:
+    weight: 2
+    label: above
+    settings:
+      format_type: medium
+      timezone_override: ''
+    third_party_settings: {  }
+    type: datetime_default
+    region: content
+  field_entrance_fee:
```

Comment:
Could be field_registration_entrance_fee

Reply:
Could be. But it probably should be discussed with @nagyad .

Reply:
It's fine `field_entrance_fee`